src/makeRoutes.js

```diff
@@ -1,12 +1,14 @@
 import userRouter from './routes/user';
 import userAuthRouter from './routes/user/auth';
+import EmailController from './controllers/EmailController';
```

Comment:
This should be importing the Email router like we are for Users, not the controller.

Reply:
done